Samples/Samples/View/BrowserPage.xaml

```diff
@@ -17,9 +17,9 @@
             <Label Text="Browser Launch Type:"/>
             <Picker HorizontalOptions="FillAndExpand"
                     ItemsSource="{Binding BrowserLaunchTypes}" 
-                    SelectedIndex="{Binding BrowserType, Mode=TwoWay}" />
+                    SelectedItem="{Binding BrowserType, Mode=TwoWay}" />
```

Comment:
Isn't the items source `BrowserLaunchTypes` is a list of strings? You appear to be mapping it to the `BrowserType` property which is an int, and you are using the `SelectedItem` property - won't this be mapping a string to an int?